claude_mcp.sh

```
@@ -314,38 +320,38 @@ display_step() {
 # Function to wait for parallel checks and collect results
 collect_parallel_results() {
     local timeout_seconds=10
-    
+
     for name in "${!PARALLEL_PIDS[@]}"; do
         local pid=${PARALLEL_PIDS["$name"]}
         local check_file="/tmp/mcp_check_$name.status"
-        
+
         # Each process gets individual timeout calculation
         local process_start_time=$(date +%s)
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)